src/components/WebSocketBannerAlert/index.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2018-2020 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import { Banner } from '@patternfly/react-core';
+import React from 'react';
+import { connect, ConnectedProps } from 'react-redux';
+import { container } from '../../inversify.config';
+import { CheWorkspaceClient } from '../../services/cheWorkspaceClient';
+import { AppState } from '../../store';
+
+type Props = MappedProps & {};
+
+type State = {
+  erroringWebSockets: string[];
+};
+
+class WebSocketBannerAlert extends React.PureComponent<Props, State> {
+  private readonly cheWorkspaceClient: CheWorkspaceClient;
+
+  constructor(props: Props) {
+    super(props);
+    this.cheWorkspaceClient = container.get(CheWorkspaceClient);
+    this.state = {
+      erroringWebSockets: this.cheWorkspaceClient.failingWebSockets
+    };
+  }
+
+  public componentWillUnmount() {
+    this.cheWorkspaceClient.removeWebSocketFailedListener();
+  }
+
+  public componentDidMount() {
+    this.cheWorkspaceClient.onWebSocketFailed(() => {
+      this.setState({
+        erroringWebSockets: this.cheWorkspaceClient.failingWebSockets
+      });
+    });
+  }
+
+  render() {
+    if (this.state.erroringWebSockets.length === 0) {
+      return null;
+    }
+
+    const erroringWebSockets = this.state.erroringWebSockets.join(', ');
+    const webSocketTroubleshootingDocs = this.props.brandingStore.data.docs.webSocketTroubleshooting;
+    return (
+      <Banner className="pf-u-text-align-center" variant="warning">
+        WebSocket connections &quot;{erroringWebSockets}&quot; are failing
```

Comment:
When I use not full screen, the error is cut with no way to check the full version, maybe we need to rethink our message and put the link at the beginning
![Screenshot_20210108_144523](https://user-images.githubusercontent.com/5887312/104017094-2918e480-51c0-11eb-82df-703cde94c4fb.png)
WebSocket connection is failed, please refer to the Network Troubleshooting section of the user guide to check if it happens due to network restrictions

Reply:
^ @ParvathyVR WDYT?

Reply:
Hi @sleshchenko! It's a limiting that he [PF banner component](https://www.patternfly.org/v4/components/banner/react-demos) truncates the text for smaller screen sizes and there is no way to see the full text. May be this is something that I can bring to the notice of the PF team.
But meanwhile,i agree that we could use a shorter version of the message as a workaround.

Reply:
Hi @sleshchenko These are the messages suggestions:
WebSocket connection has failed. Refer to [Network Troubleshooting](url) in the user guide to check for network restrictions.
 **Shorter version:** WebSocket connection has failed. Refer to [Network Troubleshooting](url) in the user guide.